uPortal-portlets/src/main/java/org/apereo/portal/portlets/portletadmin/PortletAdministrationHelper.java

```diff
@@ -268,7 +270,9 @@ private PortletDefinitionForm createNewPortletDefinitionForm() {
      */
     public PortletDefinitionForm savePortletRegistration(
             IPerson publisher, PortletDefinitionForm form) throws Exception {
-
+        if (this.logger.isInfoEnabled()) {
+            logger.info("In savePortletRegistration() - for: " + form.getPortletName());
```

Comment:
Consider instead `trace` level logging for a log message about, well, tracing path through the code.

Reply:
Good point.  Switched several of the logging statements to trace.